src/trusted/service_runtime/sel_ldr.c

```diff
@@ -1731,7 +1731,7 @@ void NaClCopyExecutionContext(struct NaClApp *nap_parent, struct NaClApp *nap_ch
   NaClLog(2, "Initializing arch switcher\n");
   NaClInitSwitchToApp(nap_child);
   NaClLog(2, "Installing trampoline\n");
-  NaClLoadTrampoline(nap_child);
+  //NaClLoadTrampoline(nap_child); handled in NaClLoadFileAslr
```

Comment:
Should this still be commented?

Reply:
I think so, as I'm not entirely sure whether we'll want to stop going through most of NaClLoadFileAslr in forks or not, as it does certain things twice that we can't remove so easily here.

Reply:
Still commented?

Reply:
Yep. I think that it makes sense as we probably want to not use most of NaClLoadFileAslr on fork and therefore we will probably want to uncomment this line in the not too distant future.